user_comments/contrib/is_core/cores.py

```diff
@@ -40,9 +41,20 @@ def get_objects(self):
         )
 
 
+class DetailCommentModelFormView(DetailModelFormView):
+
+    def get_readonly_fields(self):
+        return (
+            tuple(field for field in super().generate_fields() if field !='comment')
+            if (hasattr(self.core, 'can_update_only_comment') and self.core.can_update_only_comment == True)
+            else super().generate_fields()
+        )
+
+
 class CommentISCoreMixin:
```

Comment:
where is property can_update_only_comment?

Reply:
maybe is should be method?

Reply:
I am not sure what you mean exactly. I didn't want to add the property `can_update_only_comment` to the mixin, since there are legitimate cases where you want to allow update of the whole object/form etc.

Adding it here as `can_update_only_comment = True` would mean that it would have to be defined as `False` on all cores/views where we want to allow update of the whole object. Possible, sure, but I don'ŧ really see any advantage to that, apart from enforcing the usage of the property.

Why method?

Reply:
The way I meant it is that the property `can_update_only_comment` is added to the relevant core.

Reply:
Solved by discussion elsewhere.